k8s/helm/templates/locust-job.yaml

```diff
@@ -31,6 +31,8 @@ spec:
               value: "{{ .Values.host }}"
```

Comment:
Slightly unrelated to this change but `image: eu.gcr.io/census-eq-ci/eq-survey-runner-benchmark:latest` is wrong and is worth updating while we are here. Pipelines run fine since it overrides it. Worth pointing this to our dev gar. There is reference in the README to census gcr also that should be updated.